cgen/__init__.py

```diff
@@ -26,6 +26,18 @@
 
 import numpy
 from pytools import memoize_method, memoize
+import warnings
+
+from cgen.preprocessor import Generable, Block, Collection, Comment, Line
+from cgen.preprocessor import Define as _Define
+from cgen.preprocessor import Include as _Include
+from cgen.preprocessor import Pragma as _Pragma
+from cgen.preprocessor import IfDef as _IfDef
+from cgen.preprocessor import IfNDef as _IfNDef
```

Comment:
How come this is not a single, comma-separated import?

Reply:
Right, I will update the import statement as:
```python
from cgen.preprocessor import (
    Define as _Define,
    Include as _Include,
    Pragma as _Pragma,
    IfDef as _IfDef,
    IfNDef as _IfNDef)
```